include/machine.h

```diff
@@ -26,6 +26,15 @@ static inline paddr_t CONST addrFromPPtr(void *pptr)
     return (paddr_t)pptr - PPTR_BASE_OFFSET;
 }
 
+/* When obtaining a physical address from a reference to address from the
+ * kernel ELF mapping, this function must be used. */
+static inline paddr_t CONST addrFromKPPtr(void *pptr)
```

Comment:
add a "p" in the name (`paddrFromKPPtr()`) to clearly imply this is a physical address? That make the code that uses this easier to understand.

Reply:
Whilst I agree, I don't want to make an edit that would touch that many locations here.